src/OpenTelemetry/OpenTelemetryInvocationContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.OpenTelemetry
+{
+    internal class OpenTelemetryInvocationContext
+    {
+        public OpenTelemetryInvocationContext(string invocationId, string traceParent, string traceState)
+        {
+            InvocationId = invocationId;
+            TraceParent = traceParent;
+            TraceState = traceState;
+        }
+
+        public bool isValid()
```

Comment:
Please use PascalCase convention for naming methods.

Reply:
Done